### PR TITLE
Renamed step, limited 'PAO must gather' to 4.13

### PR DESCRIPTION
**Pull request by Sandeepyadav93**
PAO must gather is included in OCP must gather
starting 4.14, so limiting execution of PAO must
gather till 4.13.

Also, renamding the step because of above changes.

[CNF-10817](https://issues.redhat.com//browse/CNF-10817)

### CODE REVIEW

**Comment by Sandeepyadav93** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-telco5g-cnftests

File: ci-operator/step-registry/telco5g/log-gather/telco5g-log-gather-commands.sh [hunk 1]
```diff
@@ -82,33 +81,26 @@ for node in $(oc get node -oname | grep cnf); do
...
+echo "************ telco5g gather-pao commands ************"
+if [[ "$T5CI_VERSION" == "4.11" || "$T5CI_VERSION" == "4.12" || "$T5CI_VERSION" == "4.13" ]]; then
+    echo "Running gather-pao for T5CI_VERSION=${T5CI_VERSION}"
+
+    if [[ "$T5CI_VERSION" == "4.11" ]]; then
+        pao_mg_tag="4.11"
+    elif [[ "$T5CI_VERSION" == "4.12" ]]; then
+        pao_mg_tag="4.12"
+    elif [[ "$T5CI_VERSION" == "4.13" ]]; then
+        pao_mg_tag="4.13"
```
Comment (sshnaidm) [inline ci-operator/step-registry/telco5g/log-gather/telco5g-log-gather-commands.sh:93]: Looks like it's consistent here? Maybe then just
```bash
pao_mg_tag="$T5CI_VERSION"
```

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-sno-cnftests periodic-ci-openshift-release-master-nightly-4.11-e2e-telco5g-cnftests

File: ci-operator/step-registry/telco5g/log-gather/telco5g-log-gather-commands.sh (hunk 1 above)
Comment (Sandeepyadav93) [reply ci-operator/step-registry/telco5g/log-gather/telco5g-log-gather-commands.sh:93]: yeah, good point, I will update after pj rehearse job passes.

**Comment by Sandeepyadav93** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g periodic-ci-openshift-release-master-nightly-4.11-e2e-telco5g-cnftests periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-sno-cnftests

**Pull request merged**